packages/components/skeleton/src/SkeletonText/SkeletonText.tsx

```diff
@@ -23,6 +24,14 @@ export interface SkeletonTextProps {
    * Spacing between lines (in pixels)
    */
   marginBottom?: stringOrNumber;
+  /**
+   * X-axis border radius (in number)
+   */
+  radiusX?: stringOrNumber;
+  /**
+   * Y-axis border radius (in number)
+   */
+  radiusY?: stringOrNumber;
```

Comment:
is there a use case for a different border-radius between X and Y. Can we just expose one property that will be used for both values?

Reply:
ah, I see that we already had it as a separate props in the SkeletonImage